lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -607,6 +607,23 @@ def transaction_hash(result)
           'issuing_bank'        => transaction.credit_card_details.issuing_bank
         }
 
+        network_token_details = {
+          'debit'               => transaction.network_token_details.debit,
+          'prepaid'             => transaction.network_token_details.prepaid,
+          'issuing_bank'        => transaction.network_token_details.issuing_bank
+        }
+
+        google_pay_details = {
```

Comment:
@yunnydang Any reason why no `issuing_bank` in this block? I notice it's omitted from the test assertions too.

Reply:
Hye @jcreiff ! According to their [docs](https://developer.paypal.com/braintree/docs/reference/response/transaction/ruby#:~:text=of%20the%20request.-,google_pay_details,-obj), BT does not return the issuing_bank field for google pay transactions, hoping this helps!